testing/go/create_function_test.go

```diff
@@ -168,6 +168,29 @@ $$ LANGUAGE plpgsql;`},
 				},
 			},
 		},
+		{
+			// TODO: Returning an integer seems to not be supported yet?
```

Comment:
This should 100% be supported, there shouldn't be any distinction in the code for which return types are valid or invalid. We should look into this and see why it's failing

Reply:
Yup yup, just pushed a new commit that fixes integer return types.